hack/cluster/deploy-cluster.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+az login
+echo "Put name of resource to be created"
+read resrc
+echo "Enter location"
+read location
+az group create --name $resrc --location $location
+echo "Paste your subscription id here"
+read id2
+echo "Type the filename of the generated api-model to be used in .json format"
+read filename
```

Comment:
let's merge #140 here and set the path to it.

Reply:
Okay Sir

Reply:
Sir there are two ways of doing it, either we can change the directory of the file hence merge the #140 or we can link it by assuming that the file is in 
`../tests/k8s-azure/manifests/`